test/fluree/db/query/negation_test.clj

```diff
@@ -0,0 +1,46 @@
+(ns fluree.db.query.negation-test
+  (:require  [clojure.test :as t :refer [deftest testing is]]
+             [fluree.db.json-ld.api :as fluree]
+             [fluree.db.test-utils :as test-utils]))
+
+(deftest minus
```

Comment:
This tests needs more commentary. It's easier on people coming to the test later if you describe, in English, what the setup code is doing and *why* the queries should return what they do. `clojure.test` includes facilities to give that commentary with the `testing` macro and the second argument to the `is` macro. We should use them more.